src/js/page.js

```diff
@@ -65,6 +66,11 @@ export default {
                 this._setLangChangeHandlers();
                 this.active = true;
                 this._flipToFirst();
+
+                const nextPageButton = document.querySelector( '.next-page' );
```

Comment:
see $btnNext a few lines above

Reply:
`$btnNext` seemed to be missing the `setAttribute` method

Reply:
going to use `attr` method